hw_5.py

```diff
@@ -0,0 +1,158 @@
+import re
+import pandas as pd
+
+
+"""Из текстового файла удалить все слова, содержащие от трех до пяти символов, но при этом из каждой строки должно быть
+удалено только четное количество таких слов.
+"""
+
+
+class WordParsing(object):
+    """Class for opening text file and delete certain words."""
+
+    def __init__(self, file_to_open):
+        self._file_to_open = file_to_open
+        self._oper_string = ''
+
+    def open_file(self):
+        """Method to read from existing file."""
+        with open(self._file_to_open, 'r') as file_to_read:
```

Comment:
Here could be `FileNotFoundError` error. So would be nice to check before trying to open it.